config/module.go

```diff
@@ -272,7 +276,7 @@ func (m *Module) FirstRun(
 	var pathErr *os.PathError
 	switch {
 	case errors.As(err, &pathErr):
-		logger.Debugw("meta.json not found, skipping first run", "error", err)
+		logger.Infow("meta.json does not exist, skipping first run")
```

Comment:
I waffled on whether this case should also be writing out a "first run succeeded" file. I opted not to -- just in case we ever had a bug with `getJSONManifest` incorrectly missing a file that it should be seeing.

Reply:
was wrapping errors how we failed this check?

Reply:
I'm not sure I follow. This change is just about the message we print:

`meta.json not found` sounds like it should* have been found. Especially if its accompanied by an error + stacks. I wanted to soften the words such that lacking a `meta.json` is an ok state. And not bother with printing the error. Not existing _is_ "the error".

Reply:
ah, I thought I was seeing the `failed to parse` error when meta.json was not found, so wondering if this PR fixed it by not wrapping the errors